Dockerfile

```diff
@@ -1,35 +1,36 @@
 # Pull base image
-FROM python:3.6-alpine
+FROM helsinkitest/python-node:3.6-8-slim
 
 # Set environment varibles
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-# Install packages
-RUN apk add --update --no-cache --virtual .runtime-deps \
-    jpeg-dev \
-    gettext \
-    nodejs \
-    nodejs-npm
-
 # Set work directory
-WORKDIR /code
+WORKDIR /app
 
-# Install dependencies
+# Install packages
 COPY requirements.txt .
-RUN apk add --no-cache --virtual .build-deps build-base git libxslt-dev zlib-dev postgresql-dev \
-    && git config --global http.postBuffer 524288000 \
-    && pip install --upgrade pip \
-    && pip install --no-cache-dir -r requirements.txt --src /usr/local/src \
-    && apk add --virtual .runtime-deps postgresql-client \
-    && apk del .build-deps
+RUN apt-get update \
```

Comment:
While this is ok and works as expected, if you want to write less there is a helper function in the image as well :)
```
RUN apt-install.sh \
    build-essential \
    gettext \
    libjpeg-dev \
    libpq-dev \
    netcat \
&& pip install --no-cache-dir -r requirements.txt --src /usr/local/src \
&& apt-cleanup.sh build-essential
```

Reply:
Hmm.. nice, that is a smart move to use scripts and if ever needed, there can add also additional packages. Don't know yet what for but it's possible 😄 This is now changed to use scripts.